upload/upload.go

```diff
@@ -174,6 +174,17 @@ func uploadFiles(files, outFiles []string, targetDir string, config *Config) err
 		if err != nil {
 			return err
 		}
+		// Check if the file is encrypted and warn if not
+		// Extracting the first 8 bytes of the header - crypt4gh
+		magicWord := make([]byte, 8)
+		_, err = f.Read(magicWord)
+		if err != nil {
+			fmt.Printf("error reading input file %s, reason: %v", filename, err)
+		}
+		if string(magicWord) != "crypt4gh" {
+			fmt.Printf("input file %s is not encrypted", filename)
```

Comment:
I'd prefer that all files are checked before any uploads start, and that upload (of anything) only starts if something like the `continue` flag from the encrypt module is given. Otherwise there's a risk that a lot of partial unencrypted files are uploaded! :)